### DISCUSSION TITLE
Trigger manual flows on users (directus_users)

### DISCUSSION THREAD

**Discussion by vwasteels** [directus/directus#20648] [Feature Requests]
### Summary

Hello, 

I'd like to be able to trigger a manual flow on a user, from the collection and from the itempage.

I don't think it's possible right now, or did I miss something ?

Thank you

### Basic Example

for example : send a system email to all users with a specific role

### Motivation

add possibility to trigger flow manually on users

### Detailed Design

exactly the same design than the actual manual trigger flow

### Requirements List

take exactly same behaviour than existing manual trigger, but on users

### Drawbacks

I can't imagine why

### Alternatives

for now, I have put the trigger on another collection, and use the require confirmation feature to add a collection field to look for a specific directus_users. It's a trick, not elegant, and I can't select more than one user.

### Adoption Strategy

not a breaking change

**Comment by spectaA** [comment]
I'm looking for the same feature too 👍

**Comment by moseskamau338** [comment]
How did this go?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
